### DISCUSSION TITLE
刚开始玩剑与远征启程，真的是太上头啦！

### DISCUSSION THREAD

**Discussion by kallen1314** [share121/inter-knot#1292] [其他]
一开始觉得画风好看，开服福利给的也多，一开始确实上头，但一卡关了那是路边狗都打不过。给大家看我目前最喜欢的角色，真好看啊她

**Comment by lllliufeng** [comment]
没传好呀兄弟

**Reply by lllliufeng** [reply]
要等uploading这词消失

**Reply by kallen1314** [reply]
哦哦，这就重新传

**Comment by kallen1314** [comment]
![Image_1723349903277](https://github.com/user-attachments/assets/a39da007-d3cc-4d41-8b8d-249f2a5ef60a)

**Comment by Rolmans** [comment]
这玩意真的好玩吗？
